docs/cli/geo/maps.md

```diff
@@ -58,7 +58,7 @@ Amazon Location Service offers three pricing plans namely, `RequestBasedUsage`,
 You select one of the following pricing plans for the Amazon Location Service resources you create for your application.
 Amplify helps you determine the best pricing plan for you by guiding you through a set of questions.
 These pricing plan related set of questions are presented to you when you add the first Geo resource to your application. 
-Once you chose the pricing plan, it will be automatically used for the subsequent Geo resources added to your application.
+Once you chose the pricing plan, it will be automatically used for the subsequent Geo resources added to your application. Review [Amazon Location Service Pricing](https://aws.amazon.com/location/pricing/) for more details.
```

Comment:
Link to pricing plan doc is already included ~ [line 95-96](https://github.com/aws-amplify/docs/pull/3506/files#diff-7f088c60496cbfa52d069e93a7cb8f559c97e4a159dcfc5a7c9bda704ffd8d1fR95).

Reply:
That's interesting, @hdworld11 can you check what the requirements are then?